lib/rules/selector-max-id/index.mjs

```diff
@@ -106,13 +108,13 @@ const rule = (primary, secondaryOptions) => {
 				return;
 			}
 
-			for (const selector of ruleNode.selectors) {
-				for (const resolvedSelector of resolvedNestedSelector(selector, ruleNode)) {
-					parseSelector(resolvedSelector, result, ruleNode, (container) =>
-						checkSelector(container, ruleNode),
-					);
-				}
-			}
+			if (!isStandardSyntaxSelector(ruleNode.selector)) return;
```

Comment:
nit
style

either both `return;` or both `{ return; }`

Reply:
Good catch, thank you!